pyproject.toml

```
@@ -1,2 +1,69 @@
 [build-system]
-requires = ['setuptools', 'wheel', 'Cython', 'oldest-supported-numpy']
+requires = [
+    "setuptools",
```

```suggestion
    "setuptools>=61.0",
```
Maybe this version is old enough that we can assume it's presence.  v61 was the version where they deployed the current `pyproject` behaviour that this all depends on,